### DISCUSSION TITLE
Automatic Backup to Google Cloud

### DISCUSSION THREAD

**Discussion by Navid200** [NightscoutFoundation/xDrip#2314] [Show and tell]
Starting with today's Nightly release (2022.09.08b, not 2022.09.08), you can perform automatic backups to Google Drive.  
[Google Cloud Backup](https://navid200.github.io/xDrip/docs/GoogleDriveBackup.html)

However, this is a new feature that requires testing. It is possible that there may be malfunctions caused by this in xDrip Sync or Wear integration.  
It could malfunction and that's why it is categorized as a Nightly (pre-release).   Please backup with the existing mechanism before testing the new restore feature.   
Please provide feedback and we will fix any malfunctions.  

Thanks

**Comment by bmpreston** [comment]
Any updates to this?  The link is broken.  Thanks!

**Reply by Navid200** [reply]
Thanks for letting me know.  The link is now to the correct page.
Yes, it works.  Please let me know if you run into any trouble.
